postgres/tests/test_statements.py

```diff
@@ -750,6 +750,7 @@ def test_statement_run_explain_errors(
 
     check.check(dbm_instance)
     _, explain_err_code, err = check.statement_samples._run_explain_safe("datadog_test", query, query, query)
+    check.check(dbm_instance)
```

Comment:
Why do you run this check again?

Reply:
Because the size of the cache is only reported at the end of each check run.